### PR TITLE
chapter 1 solutions

### PR DESCRIPTION
**Pull request by pkk**
### Solutions for Chapter 1

cc @vrom911 @chshersh

### CODE REVIEW

File: src/Chapter1.hs [hunk 1]
```diff
@@ -568,7 +575,10 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c 
+      | (c == 'a' || c == 'e' || c == 'i' || c == 'o' ||  c == 'u') = True
+      | otherwise = False
```
Comment (vrom911) [inline src/Chapter1.hs:581]: Nice one! 👍🏼 
As `==` and `||` return `Bool` itself, you can just return the result of this function:

```haskell
isVowel c = c == 'a' || c == 'e' || c == 'i' || c == 'o' ||  c == 'u'
```

File: src/Chapter1.hs [hunk 2]
```diff
@@ -632,7 +642,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = a + b
+  where
+    a = quot (mod (abs n) 100) 10
+    b = mod (abs n) 10
```
Comment (vrom911) [inline src/Chapter1.hs:649]: Usually `quot` is paired with `rem`, while `mod` with `div`, these pair are comprehensive to each other 🙂

**Review by vrom911 — APPROVED** [review]
Awesome work on solving the challenges! ⭐ 
Congrats with finishing Chapter One 🥳 

And well done with the Advanced Task :medal_sports:

File: src/Chapter1.hs (hunk 1 above)
Comment (pkk) [reply src/Chapter1.hs:581]: Thanks for this elegant and simple alternative

**Pull request merged**